visidata/guides/MenuGuide.md

```diff
@@ -0,0 +1,24 @@
+# The Menu System
+
+Located at the very top of the screen, the menu organizes many commands within submenu trees.
+
+Navigate the menu like a GUI by using the mouse to click on menu items.
+
+To navigate with the keyboard, `Alt+H` will open the **Help** menu, and from there use the arrow keys to move between submenus.
```

Comment:
Similarly, "Alt+H to open the Help menu" and "Arrow keys to move between submenus", probably on a separate line.